2nd-gen/alerts-to-discord/functions/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { onNewFatalIssuePublished } = require("firebase-functions/v2/alerts");
+const logger = require("firebase-functions/logger");
+
+const fetch = require("node-fetch");
+
```

Comment:
Then maybe // [START discordMessageTrigger]  here.

Should we tag all the way down through the error?

Reply:
Added tags!